gslab_scons/size_warning.py

```diff
@@ -210,4 +200,14 @@ def check_track_lfs(filepath, attrib_path = '../.gitattributes'):
         return False
     return True
 
+def add_to_lfs(filepath, attrib_path = '../.gitattributes'):
+    '''
+    This function tracks existing files with git lfs by writing down 
+    the given file path to the git attribute file (the default path is '../.gitattributes')
```

Comment:
I think the words in the bracket can be deleted, and change "git attribute" to ".gitattribute"